nss-day-cohort-30/bangazon-site-suave-salmon
# Aj

Fixes:
-
-

Changes proposed in this request & reasons behind organizational or architectural decisions:
- Profile section completed. Users can access their profile and view payment options, create payment method and view order history.
-

Steps to test:
- Select the profile link in the navigation bar. Choose your options to view orders, options or create payment method. 
-
-
-

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-
-

Commit message:
-  [AJ 2d53fa8] user can view profile, add payment, view orders and see payment options...would like to allow user to  edit profile info but not required by ticket
--




Link to feature ticket:
- http://localhost:5000/ApplicationUsers 

@suave-salmon
